usaspending_api/references/migrations/0060_auto_20170330_1522.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-03-30 15:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('references', '0059_auto_20170330_1426'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='refprogramactivity',
+            name='ref_program_activity_id',
```

Comment:
Happy to say goodbye to `ref_program_activity_id`. I know this type of work is tedious--thank for you the housekeeping.